pyspark-notebook/Dockerfile

```diff
@@ -6,8 +6,6 @@ FROM $BASE_CONTAINER
 
 LABEL maintainer="Jupyter Project <jupyter@googlegroups.com>"
 
-# Fix: https://github.com/hadolint/hadolint/wiki/DL4006
-# Fix: https://github.com/koalaman/shellcheck/wiki/SC3014
```

Comment:
This is unrelated.
Please, don't make unrelated changes, to make your PR easier and more focused.

Reply:
I don`t know why this came up. but it`s gone now.

Reply:
I still see this change, please, add these lines back.